pkg/rbac/rego.go

```diff
@@ -0,0 +1,71 @@
+package rbac
+
+import (
+	"context"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/rego"
+	"github.com/pkg/errors"
+)
+
+var regoModule string = `
```

Comment:
would be nice to have this in a separate .rego file

Reply:
then we will have to use gobindata. we can do that later if necessary